src/styles/_component.scss

```diff
@@ -0,0 +1,2 @@
+@import "./reset";
```

Comment:
A partial scss file just for 2 lines of code seems unnecessary. Import `./reset` in `style.scss`  should be enough, no?

Reply:
I disagree. The individual components shouldn't have the styles from `style.scss` but they _should_ have the styles from `reset.scss` and `variables.scss`. So instead of writing two imports on every component, we only need to write one. Plus if we add another library or utility scss file we want in every component, we only need to add it to one place. Instead of every single other component.

Reply:
If `reset.scss` is imported into `style.scss`, we don't need to import it on components anymore, no? I haven't checked, but if that works, we wouldn't need the `component.scss`

Reply:
I see what you're saying. I'll check it later. The variables are definitely not passed to components. I just assumed the reset styles wouldn't be either.